src/app/api/admin/analytics/revenue/route.ts

```
@@ -0,0 +1,277 @@
+            dateStr = currentDate.toISOString().split("T")[0];
+            existingData = data.find((item) => item._id === dateStr);
+            result.push({
+              date: dateStr,
+              amount: existingData ? (existingData[valueField] as number) : 0,
+            });
+            currentDate.setDate(currentDate.getDate() + 1);
+            break;
+
+          case "weekly":
+            // Get week number in ISO format (YYYY-Www)
+            const year = currentDate.getFullYear();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)